lessons/pydata/classification_metrics/index.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Tady by to dle mého chtělo více vysvětlit a třeba dodat i příklad. Ten se psy na fotografii z wiki, odkud je nejspíše inspirována i kresba, by mohl posloužit dobře: https://en.wikipedia.org/wiki/Precision_and_recall</p><p> </p><p>Stejně jako by to možná chělo popis k obrázku. Něco jako: To co je v kolečku označil náš klasifikátor jako pozitivní případ, i když se v několika případech spletl.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>